#!/usr/bin/env python
PACKAGE = "my_pkg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("Kp",    double_t,    0, "Kp in PID", 0.017,  0, 10)
gen.add("Ki",    double_t,    0, "Ki in PID", 0,  0, 10)
gen.add("Kd",    double_t,    0, "Kd in PID", 0.35,  0, 10)
gen.add("ServoBias", int_t , 0, "Servo Bias",0,-600,600)
gen.add("p_vel",    double_t,    0, "Purpose velocity(m/s)", -10,  0, 10)
gen.add("p_ang",    double_t,    0, "Purpose angular velocity('/s)", -10,  0, 10)
gen.add("servo_angle",    double_t,    0, "Purpose Servo agnle(')", 0,  -75, 75)

exit(gen.generate(PACKAGE, "my_pkg", "PID_reconfig"))
